python/mathsys/dev/main/latex.py

```
@@ -223,38 +251,39 @@ def __str__(self) -> str:
 class LTXInfinite:
     def __str__(self) -> str:
         return r"\infty "
 
 #> 5ºLEVEL -> VARIABLE
-@dataclass(frozen = True)
+@dataclass
 class LTXVariable:
     name: str
     def __str__(self) -> str:
         curated = self.name
         for source, replace in VARIABLES.items(): curated = curated.replace(source, replace)
```

Ruff (0.14.5)

Multiple statements on one line (colon)

(E701)